codegen/module_system.go

```diff
@@ -58,6 +58,16 @@ func NewDefaultModuleSystem(h *PackageHelper) (*module.System, error) {
 		)
 	}
 
+	if err := system.RegisterClassType("client", "tchannel", &TCahnnelClientGenerator{
```

Comment:
fix in https://github.com/uber/zanzibar/pull/57/commits/bb184bdb681025e1c171837b840b81c89ad5f9f8